plugins/modules/provision_workflow_manager.py

```diff
@@ -102,24 +116,29 @@
             description:
               - If set to 'true', Access Point (AP) provisioning will be skipped during the workflow.
               - Use this option when AP provisioning is not required as part of the current operation.
+              - Supported in Cisco Catalyst version 2.3.7.6 and later.
             type: bool
             default: false
         rolling_ap_upgrade:
             description:
               - Configuration options for performing a rolling upgrade of Access Points (APs) in phases.
               - Allows control over the gradual rebooting of APs during the upgrade process.
+              - Supported in Cisco Catalyst version 2.3.7.6 and later.
             type: dict
             suboptions:
                 enable_rolling_ap_upgrade:
                     description:
                       - Enable or disable the rolling AP upgrade feature.
                       - If set to 'true', APs will be upgraded in batches based on the specified reboot percentage.
+                      - Supported in Cisco Catalyst version 2.3.7.6 and later.
                     type: bool
                     default: false
                 ap_reboot_percentage:
                     description:
                       - The percentage of APs to reboot simultaneously during an upgrade.
                       - Must be an integer value, typically between 1 and 100, indicating the proportion of APs to upgrade at a time.
+                      - Supported in Cisco Catalyst version 2.3.7.6 and later.
+                      - Permissible values are - 5, 15, 25
```

Comment:
We're stating the value between 1 and 100. But here, we're saying that the allowed values are 5, 15, 25. Is that right?  Also we specify the default value as 20.. 

If you want to stick with 5, 15, 20, and 25 as the permissible values, you could update the description like this:
```
- Must be one of the permissible values: 5, 15, 20, or 25, representing the proportion of APs to reboot at once.
 - Default value is 20.
```

Reply:
yes, Madhan the permissible values - [5, 15, 25] and I have remove the default value as 20 and updated the code